styles/all/template/push_worker.js.twig

```diff
@@ -1,3 +1,16 @@
+/**
+ * Event listener for install event
+ */
+self.addEventListener('install', () => {
+	// Call to ensure service worker is correctly updated
+	self.skipWaiting();
+});
+
+/* global clients */
+self.addEventListener('activate', event => {
+	event.waitUntil(clients.claim());
```

Comment:
It's not global but `self` inside worker.
```suggestion
	event.waitUntil(self.clients.claim());
```

Reply:
But that's not how they say to use it here https://developer.mozilla.org/en-US/docs/Web/API/Clients/claim

Reply:
Adding the comment like that does nothing as far as javascript code cares, it just _tricks_ the linter into letting it pass